deployment/fedora-package-x64/pkg-src/jellyfin.spec

```diff
@@ -13,20 +13,29 @@ Summary:        The Free Software Media Browser
 License:        GPLv2
 URL:            https://jellyfin.media
 Source0:        %{name}-%{version}.tar.gz
-Source1:        jellyfin.service
-Source2:        jellyfin.env
-Source3:        jellyfin.sudoers
-Source4:        restart.sh
-Source5:        jellyfin.override.conf
-Source6:        jellyfin-firewalld.xml
+Source1:        https://github.com/%{name}/%{name}-web/archive/%{name}-web-%{version}.tar.gz
```

Comment:
This URL will return 404 unless the archive is downloaded beforehand with the specified name e.g. `jellyfin-web-10.4.0.tar.gz` since the real filename is `v10.4.0.tar.gz`, this should be commented.

It's clear that `jellyfin-web-10.4.0.tar.gz` is far more readable than `v10.4.0.tar.gz` but it should be noted that steps like in the `.copr/Makefile` are required (Downloading from the correct URL and renaming to *jellyfin-web-xyz.tar.gz*).